roles/cron/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Set cron_paths fact
+  set_fact:
+    cron_paths: "/usr/local/bin:/usr/bin:/bin:/usr/sbin"
+
+- name: add cron locations to path
+  lineinfile:
+    dest: /etc/environment
+    state: present
+    backrefs: yes
+    regexp: "PATH=([\"]*)((?!.*?{{ cron_paths }}).*?)([\"]*)$"
```

Comment:
Why the change of environment?

Reply:
Good question, I did this month's ago, checked our current Cron role and it wasn't there, I'll remove it